docs/topic_guides/choosing_comparators.ipynb

```diff
@@ -0,0 +1,1249 @@
+    "For example, consider a single Jaro-Winkler `Comparison Level` threshold of 0.7 would lead to \"Rachael\" being considered as providing the same amount evidence for a record matching as \"iRchard\".\n",
+    "\n",
+    "An alternative way around this is to construct a `Comparison` with multiple levels, each corresponding to a different threshold of Jaro-Winkler similarity. For example, below we construct a `Comparison` using the `Comparison Library` function [jaro_winkler_at_thresholds](../comparison_level_library.md#JaroWinklerAtThresholdsComparisonBase) with multiple levels for different match thresholds.:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'output_column_name': 'first_name',\n",
+       " 'comparison_levels': [{'sql_condition': '\"first_name_l\" IS NULL OR \"first_name_r\" IS NULL',\n",
+       "   'label_for_charts': 'Null',\n",
+       "   'is_null_level': True},\n",
+       "  {'sql_condition': '\"first_name_l\" = \"first_name_r\"',\n",
+       "   'label_for_charts': 'Exact match'},\n",
+       "  {'sql_condition': 'jaro_winkler_similarity(\"first_name_l\", \"first_name_r\") >= 0.9',\n",
+       "   'label_for_charts': 'Jaro_winkler_similarity >= 0.9'},\n",
+       "  {'sql_condition': 'jaro_winkler_similarity(\"first_name_l\", \"first_name_r\") >= 0.8',\n",
+       "   'label_for_charts': 'Jaro_winkler_similarity >= 0.8'},\n",
+       "  {'sql_condition': 'jaro_winkler_similarity(\"first_name_l\", \"first_name_r\") >= 0.7',\n",
+       "   'label_for_charts': 'Jaro_winkler_similarity >= 0.7'},\n",
+       "  {'sql_condition': 'ELSE', 'label_for_charts': 'All other comparisons'}],\n",
+       " 'comparison_description': 'Exact match vs. jaro_winkler_similarity at thresholds 0.9, 0.8, 0.7 vs. anything else'}"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import splink.duckdb.duckdb_comparison_library as cl\n",
+    "\n",
+    "first_name_comparison = cl.jaro_winkler_at_thresholds(\"first_name\", [0.9, 0.8, 0.7])\n",
+    "\n",
+    "first_name_comparison.as_dict()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Where:\n",
+    "* Exact Match level will catch perfoect matches (\"Richard\").\n",
+    "* The 0.9 threshold will catch Shortenings and Typos (\"ichard\", \"Richar\", \"iRchard\", \"Richadr\",  \"Rich\").\n",
+    "* The 0.8 threshold will catch simple Nicknames/Aliases (\"Ricky\", \"Rico\").\n",
+    "* The 0.7 threshold will catch more complex Nicknames/Aliases (\"Ricky\"), but will also include less relevant names (e.g. \"Rachael\"). However, this should not be a concern as the model should give less predictive power (i.e. Match Weight) to this level of evidence.\n",
+    "* All other comparisons will end up in the \"Else\" level"
```

Comment:
I think each comparison level is meant to be on a separate line but is currently one paragraph with * between each sentence.